src/temp/ProjectListControls/index.tsx

```diff
@@ -0,0 +1,131 @@
+  const t = useTranslations('AllProjects');
+  const [isSearching, setIsSearching] = useState(false);
+  const [value, setValue] = useState(0);
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [selectedClassification, setSelectedClassification] = useState<
+    Classification[]
+  >([]);
+  const { primaryColorNew, dark } = themeProperties;
+  const isFilterApplied = selectedClassification.length > 0;
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+    setIsFilterOpen(false);
+  };
+  return (
+    <>
+      <div className={style.searchBarMainContainer}>
+        {isSearching ? (
+          <ActiveSearchField
+            setIsSearching={setIsSearching}
+            setIsFilterOpen={setIsFilterOpen}
+          />
+        ) : (
+          <div className={style.searchBarContainer}>
+            <Tabs
+              value={value}
+              onChange={handleChange}
+              TabIndicatorProps={{
+                sx: { backgroundColor: `${primaryColorNew}` },
+              }}
+            >
+              <CustomMuiTab
+                icon={
+                  <StarIcon
+                    width={'16px'}
+                    color={`${value === 1 ? dark.darkNew : primaryColorNew}`}
+                  />
+                }
+                label={
+                  <div className={style.projectLabel}>
+                    {t.rich('topProjects', {
+                      noOfProjects: topProjectCount,
+                      projectCountContainer: (chunks) => (
+                        <span className={style.projectCount}>{chunks}</span>
+                      ),
+                    })}
+                  </div>
+                }
+                sx={{ fontWeight: '700' }}
+              />
+              <CustomMuiTab
+                sx={{ fontWeight: '700' }}
+                label={
+                  <div className={style.projectLabel}>
+                    {t.rich('allProjects', {
+                      noOfProjects: projectCount,
+                      projectCountContainer: (chunks) => (
+                        <span className={style.projectCount}>{chunks}</span>
+                      ),
+                    })}
+                  </div>
+                }
+              />
+            </Tabs>
+            <div className={style.iconsContainer}>
+              <button
+                className={style.icon}
+                onClick={() => setIsSearching(!isSearching)}
+              >
+                <SearchIcon />
+              </button>
+              <div className={style.filterContainer}>
+                {isFilterApplied && <div className={style.filterIndicator} />}
+                <button
+                  className={style.icon}
+                  onClick={() => setIsFilterOpen(!isFilterOpen)}
+                >
+                  <FilterIcon width={'16px'} />
+                </button>
+              </div>
+            </div>
+          </div>
```

Comment:
1. Extract the code for tabs to a different component.
2. Why is the outer div have the `searchBarContainer` class name applied?

Reply:
2. this was an unwanted class which has been removed